eppo_metrics_sync/dbt_model_parser.py

```diff
@@ -0,0 +1,128 @@
+from collections import Counter
+from itertools import chain
+
+
+class DbtModelParser():
+
+    def __init__(self, model, dbt_model_prefix):
+        self.model = model
+        self.eppo_entities = []
+        self.eppo_timestamp = []
+        self.eppo_facts = []
+        self.eppo_properties = []
+        self.dbt_model_prefix = dbt_model_prefix
+    
+    def parse_columns(self):
+
+        for column in self.model.get('columns'):
+
+            tags = column.get('tags')
+
+            if tags:
+                column_entities = [t for t in tags if 'eppo_entity:' in t]
+
+                if column_entities:
+                    for column_entity in column_entities:
+                        column_entity_split = column_entity.split(':')
+                        if len(column_entity_split) != 2:
+                            raise ValueError(
+                                f'Invalid entity tag {column_entity} in model {self.model["name"]}'
+                            )
+                        else:
+                            self.eppo_entities.append({
+                                "entity_name": column_entity_split[1],
+                                "column": column['name']
+                            })
+                        
+
+                if 'eppo_timestamp' in tags:
+                    self.eppo_timestamp = column["name"]
+
+                if 'eppo_fact' in tags:
+                    self.eppo_facts.append({
+                        "name": column["name"],
+                        "column": column["name"],
+                        "description": column["description"]
+                    })
+                
+                if 'eppo_property' in tags:
+                    self.eppo_properties.append({
+                        "name": column["name"],
+                        "column": column["name"],
+                        "description": column["description"]
+                    })
+    
+    def validate(self):
+
+        self.validation_errors = []
```

Comment:
It looks like `self.validation_errors` is only used inside this method, so it doesn't seem necessary to make it an instance attribute. Also, instance attributes should be initialized in `__init__`